docker-images/jmxtrans/docker-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# The MIT License
```

Comment:
I'm quite curious how this works since we have ASL2.0. Anyone understands these kind of things?